### DISCUSSION TITLE
Action : build - no runners

### DISCUSSION THREAD

**Discussion by 1fini** [community/community#46319] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi guys, 

Lately, I changed my application framework (-> .net7) and the action workflow that is supposed to build and deploy the docker image on dockerhub doesn't work anymore. The build task never starts. I get the message : "waiting for workers" 
Do you guys know what could cause this?

**Comment by 1fini** [comment]
the exact message during the build step is : "Waiting for a runner to pick up this job..."

**Comment by 1fini** [comment, marked as answer]
in the workflow yml file, I chose a job with GitHub hosted executor "runs-on : Debian" but this one doesn't exist
we have to choose ubuntu-latest or other label based on ubuntu distribution

**Comment by Mw-oddan** [comment]
i want to recovery action path
